stripe/src/main/java/com/stripe/android/paymentsheet/BaseAddCardFragment.kt

```diff
@@ -305,6 +307,18 @@ internal abstract class BaseAddCardFragment(
         cardMultilineWidget.postalCodeErrorListener = null
     }
 
+    private fun populateFieldsFromNewCard() {
+        sheetViewModel.newCard?.let { (paymentMethodCreateParams, _, shouldSavePaymentMethod) ->
+            saveCardCheckbox.isChecked = shouldSavePaymentMethod
+            paymentMethodCreateParams.let { params ->
+                cardMultilineWidget.populateFromParams(params.card)
+                params.billingDetails?.let {
+                    billingAddressView.populateFromParams(it.address)
+                }
+            }
```

Comment:
nit: you can skip the `let` now and use `paymentMethodCreateParams` directly

Reply:
I am not sure if I got what you wanted here, so please just comment again if I didn't get it right.